script.js

```diff
@@ -0,0 +1,338 @@
+    }
+}
+subtract.disabled = true
+function subtracted(){
+   disableButtonRepeat()
+    if(firstDisplay.textContent !=="" && secondDisplay.textContent !== "" && signs.textContent !== ""){
+        calculate()
+        secondDisplay.textContent = firstDisplay.textContent
+        signs.textContent = "-"
+        firstDisplay.textContent = ""
+    }
+    else{
+        secondDisplayContainer.appendChild(secondDisplay);
+        secondDisplay.classList.toggle("second-display");
+        dummyDisplay.classList.remove("dummy-display");
+        secondDisplay.textContent =  firstDisplay.textContent;
+        signs.textContent = "-"
+        if(firstDisplay.textContent === "")return
+        else{
+            clearFirstDisplay()
+        }
+    }
+}
+multiply.disabled = true
+function multiplied(){
+ disableButtonRepeat()
+    if(firstDisplay.textContent !=="" && secondDisplay.textContent !== "" && signs.textContent !== ""){
+        calculate()
+        secondDisplay.textContent = firstDisplay.textContent
+        signs.textContent = "X"
+        firstDisplay.textContent = ""
+    }
+    else{
+        secondDisplayContainer.appendChild(secondDisplay);
+    secondDisplay.classList.toggle("second-display");
+    dummyDisplay.classList.remove("dummy-display");
+    secondDisplay.textContent =  firstDisplay.textContent;
+    signs.textContent = "X"
+    if(firstDisplay.textContent === "")return
+    else{
+        clearFirstDisplay()
+    } 
+    }
+}
+divide.disabled = true
+function divided(){
+    disableButtonRepeat()
+    if(firstDisplay.textContent !=="" && secondDisplay.textContent !== "" && signs.textContent !== ""){
+        calculate()
+        secondDisplay.textContent = firstDisplay.textContent
+        signs.textContent = "÷"
+        firstDisplay.textContent = ""
+    }
+    else{
+        secondDisplayContainer.appendChild(secondDisplay);
+        secondDisplay.classList.toggle("second-display");
+        dummyDisplay.classList.remove("dummy-display");
+        secondDisplay.textContent =  firstDisplay.textContent;
+        signs.textContent = "÷"
+        if(firstDisplay.textContent === "")return
+        else{
+            clearFirstDisplay()
+        }
+    }
+}
+ function operate(){
+    calculate()
+ }
+function clearFirstDisplay(){
+    firstDisplay.textContent = ""
+}
+ function calculate(){
+    let autocalculate;
+     if(signs.textContent === "+"){
+     autocalculate = +secondDisplay.textContent + +firstDisplay.textContent
+     }
+     else if(signs.textContent === "-"){
+     autocalculate = +secondDisplay.textContent - +firstDisplay.textContent
+     }
+     else if(signs.textContent === "X"){
+        autocalculate = +secondDisplay.textContent * +firstDisplay.textContent
+     }
+     else if(signs.textContent === "÷"){
+        autocalculate = +secondDisplay.textContent / +firstDisplay.textContent
+     }
+     firstDisplay.textContent = autocalculate;
+     secondDisplay.textContent = ""
+     signs.textContent = ""
+ }
```

Comment:
There seems to be some possible misbehavior when this `function` is called on pressing the `=` sign. I'll list some of them below just incase it's not the desired behaviour

To illustrate these consider a user has entered 10 * 2.5, with the result that 
```
firstDisplay = 25
secondDisplay = ''
signs = ''
```
1. If the user presses a number immediately, say 7, it gets concatenated to `firstDisplay` i.e. `firstDisplay = 257`
2. Pressing `=` simultaneously causes `firstDisplay = undefined`
3. Following up on 3, the user is not able to perform operations resulting in `NaN` 
4. User is unable to enter a decimal sign since it's still disabled from the later entry of `2.5`

Other edge cases include:
1. User presses `=` without having entered anything
2. User presses `=` without having entered a sign
3. User presses `=` without having entered a second operand
4. Allows for possible [divideByZero](https://en.wikipedia.org/wiki/Division_by_zero) leading the user into operations on Infinity